c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh

```diff
@@ -0,0 +1,180 @@
+export GYP_DEFINES="disable_nacl=1"
+
+# check out LLVM and Clang
+export CLANG_REVISION=375507
+cd $CHROMIUM_DIR
+svn checkout --force https://llvm.org/svn/llvm-project/llvm/trunk@$CLANG_REVISION llvm
+svn checkout --force https://llvm.org/svn/llvm-project/cfe/trunk@$CLANG_REVISION llvm/tools/clang
+svn checkout --force https://llvm.org/svn/llvm-project/compiler-rt/trunk@$CLANG_REVISION llvm/compiler-rt 
+
+# Build the compiler
+mkdir llvm_build && cd llvm_build
+cmake3 -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="PowerPC" -G "Unix Makefiles" ../llvm
+make -j8
+
+# Downloading the source code
+cd $CHROMIUM_DIR
+mkdir chromium && cd chromium
+gclient config --name "src" --unmanaged https://chromium.googlesource.com/chromium/src.git --custom-var="checkout_nacl=False"
+gclient sync --nohooks --no-history
+
+cd src
+git fetch https://chromium.googlesource.com/chromium/src.git +refs/tags/84.0.4118.0:chromium_84.0.4118.0
+git checkout chromium_84.0.4118.0
+
+curl -k "https://wiki.raptorcs.com/w/images/1/10/Binutils-download.py-PPC.patch" | patch -p1 third_party/binutils/download.py
+rm -rf buildtools/linux64/gn
+cp -f $CHROMIUM_DIR/gn/out/gn buildtools/linux64/gn
+# Disable gn_version_check since that overwrites power gn with amd64 gn
+sed -i '/'\''src\/buildtools\/linux64'\''/,+9d' DEPS
+gclient sync --with_branch_heads --with_tags
+
+# Patching Chromium
+cd $CHROMIUM_DIR
+git clone https://github.com/shawnanastasio/chromium_power
+cd chromium_power
+git checkout 146c60db1ac158ccb6397e3e74f7381b21a638c4
+
+sed -i '/0001-Implement-support-for-PPC64-on-Linux.patch/d' patches.json
+sed -i '/prompt_fail_action(e.reason, dest_dir, patch_path, e.output)/d' cpf.py
+sed -i '/if not DRY and e.reason != PatchFailReason.ALREADY:/d' cpf.py
+
+python3 cpf.py $CHROMIUM_DIR/chromium/src
+
+cd $CHROMIUM_DIR/chromium/src/third_party/crashpad/crashpad/
+cat ../../../../../chromium_power/crashpad/0001-Implement-support-for-PPC64-on-Linux.patch | patch -p1
+
+# Install CLANG
+cd $CHROMIUM_DIR/llvm_build/
+make install 
+
+cd $CHROMIUM_DIR/chromium/src/third_party/ffmpeg
+./chromium/scripts/build_ffmpeg.py linux ppc64
+./chromium/scripts/generate_gn.py
+./chromium/scripts/copy_config.sh
+
+cd $CHROMIUM_DIR/chromium/src/third_party/libvpx
+mkdir source/config/linux/ppc64
+./generate_gni.sh
+
+cd $CHROMIUM_DIR/chromium/src
+
+curl -k "https://wiki.raptorcs.com/w/images/b/bb/0001-sandbox-linux-seccomp-bpf-helpers-Fix-TCGETS-declara.patch" | patch -p1
+
+# Configure build parameters with gn
+mkdir out/Default
+cd out/Default
+
+echo '# Build arguments go here.
+# See "gn args <out_dir> --list" for available build arguments.
+# Release mode
+is_component_build = false
+is_debug = false
+# Disable broken features
+enable_nacl = false
+treat_warnings_as_errors = false
+enable_dav1d_decoder = false
+# For clang, add the following lines:
+is_clang = true
+clang_base_path = "/root/chromium_build/llvm_build"
```

Comment:
I believe this should be relative to CHROMIUM_DIR rather than being hard coded?

Reply:
right, I'll change that.